doc/lua_api.txt

```diff
@@ -1007,13 +1007,22 @@ The function of `param2` is determined by `paramtype2` in node definition.
       0 = y+,   1 = z+,   2 = z-,   3 = x+,   4 = x-,   5 = y-
     * facedir modulo 4 = rotation around that axis
 * `paramtype2 = "leveled"`
-    * Only valid for "nodebox" with 'type = "leveled"', and "plantlike_rooted".
+    * Only valid for "nodebox" with 'type = "leveled"' or 'type = "leveled_with_offset"',
+      and "plantlike_rooted".
         * Leveled nodebox:
             * The level of the top face of the nodebox is stored in `param2`.
             * The other faces are defined by 'fixed = {}' like 'type = "fixed"'
               nodeboxes.
             * The nodebox height is (`param2` / 64) nodes.
-            * The maximum accepted value of `param2` is 127.
+            * The maximum accepted value of `param2` is `leveled_max`. (See Node
```

Comment:
Please note that I'm not sure if this information is correct. If so, the change was forgotten in the PR that introduced this change to leveled.